test/test_linalg.py

```diff
@@ -3255,6 +3255,7 @@ def solve_test_helper(self, A_dims, b_dims, device, dtype):
         A = random_fullrank_matrix_distinct_singular_value(*A_dims, dtype=dtype).to(device)
         return b, A
 
+    @onlyOnCPUAndCUDA
```

Comment:
Skips meta device because of `RuntimeError: DispatchStub: unsupported device typemeta`.

Reply:
Seems like there are additional tests that needs to be skipped. however I can't be sure. See: 
https://app.circleci.com/pipelines/github/pytorch/pytorch/322056/workflows/fe2c23e9-fe0a-4d1f-8b55-2877c77c48eb/jobs/13459579